### DISCUSSION TITLE
11주차 과제

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#43] [Show and tell]
#과제
Data Visualization 코드
+ 스프레드시트에서 데이터 생성 -> CSV 파일
+ p5js 에서 CSV 파일 읽어서 기본 View 코드로 구현

12월 6일 밤 12시까지.
더 일찍 제출하고 많이 질문할 수록 많은 도움을 받을 수 있음!
시간 늦게 제출하면 기껏 열심히 했는데 감점! ㅠ

**Comment by eunjinhong** [comment]
홍은진
https://blog.naver.com/tnekf1012/222588490940

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222588527720

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222588523672

**Reply by youngsangcho** [reply]
수업 시간에 언급했던 잔량과 잠수시간 관계는,
잔량 x 시간의 값을 표현하면
잠수시간이 길 때의 잔량이 잠수시간의 짧을 때의 잔량과 구분되니
잔량 절대치를 표현하는 것보다 상대적인 비교가 가능하겠다는 생각이었는데,
스쿠버 다이버 입장에서는 꼭 그렇지 않을 수 있겠다는 생각도 듭니다.
잔량의 절대적인 표시로 충분할 수도 있겠죠.
김현정 학생이 생각하기에 좋은 방법으로 진행하면 됩니다-

**Comment by Joeun410** [comment]
김조은
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222588573533&categoryNo=25&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=45038

**Comment by imyeeunok** [comment]
김예은
https://blog.naver.com/010312kye/222588674853

**Reply by youngsangcho** [reply]
과제에서 사진의 복잡도를 수치화하는데에는 기존 Processing 을 사용해봐도 좋을 거 같습니다.
1. processing.org 의 3.5.4 버젼을 설치하고, 
2. Sketch - Import Library - Add Library - 에서 openCV for Processing 를 설치하세요.
3. File - Examples - Contributed Libraries - OpenCV for Processing 에서 Find Contours 를 선택

실행하면 found 166 contours 라고 뜨는데, test.jpg 를 검사를 원하는 이미지로 바꿔보고 166 숫자가 복잡도와 비슷하게 증가/감소하는지 살펴본 후 맞다면 그 값을 사용하면 됩니다.

**Comment by jwejwepark** [comment]
https://blog.naver.com/glqkfl0726/222588798938
